src/components/portal/portal.tsx

```diff
@@ -45,7 +45,7 @@ export class Portal {
      * Decides which direction the portal content should open.
      */
     @Prop()
-    public openDirection: OpenDirection = 'right';
+    public openDirection: OpenDirection = 'bottom';
```

Comment:
Shouldn't this be either `bottom-end` or `bottom-start`?

Reply:
we decided to default the location to bottom and align to center. This is also what popper.js does as default. Things look better this way if you as us :)

Reply:
Yeah, ok, I guess `bottom` is common to use on mobile devices. (Which is a little weird, since it means you will cover the newly opened menu with your hand until you move your finger away 🤦‍♂️)

Funny how I was thinking "No one uses `bottom` for menus!" just because I happen to be seated in front of a laptop right now 😂

Reply:
Eh, sorry, I was confused. No, popover isn't private, but I meant _portal_ (those two are so damn confusing… 😖).

Since `limel-portal` is private, it doesn't make sense to document breaking changes on it. The changes need to be documented on the components that people can use 😄

I'm working on it now, and I'll fix some nice example while I'm at it, so should be ready for this to be merged fairly soon. Hopefully today…